code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -29,28 +28,21 @@ Pipelines + Other Objects -> Pipe network
 	var/pipe_color
 	var/obj/item/pipe/stored
 	var/image/pipe_image
-	var/global/datum/pipe_icon_manager/icon_manager

```

Comment:
I have this great idea. Let's make a global datum, attached to all machines and when it initializes, it'll make a NEW icon manager datum, then if it already exists, we'll just return. This totally makes sense! Also, if you happen to delete it, it'll just kill the server with the lag it generates not to mention generate hundreds of runtimes!

Reply:
Absolute stroke of genius